### PR TITLE
[OCPBUGS-42690] -Updated $ sign at the starting of the commands

### PR DESCRIPTION
**Pull request by PoornimaSingour**
Updated $ sign at the starting of the commands

Version(s):

4.14, 4.15, 4.15, 4.16

Issue:

https://issues.redhat.com/browse/OCPBUGS-42690

Link to docs preview:
[Deleting a hosted cluster from your source management cluster](https://82834--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp_high_availability/hcp-disaster-recovery-aws.html#dr-hosted-cluster-within-aws-region-delete_hcp-disaster-recovery-aws)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/dr-hosted-cluster-within-aws-region-delete.adoc [hunk 1]
```diff
@@ -69,8 +69,8 @@ oc delete machineset -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME} --all || true
 ----
 # Cluster
 C_NAME=$(oc get cluster -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME} -o name)
```
Comment (mburke5678) [inline modules/dr-hosted-cluster-within-aws-region-delete.adoc:71]:
```suggestion
$ C_NAME=$(oc get cluster -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME} -o name)
```

File: modules/dr-hosted-cluster-within-aws-region-delete.adoc [hunk 2]
```diff
@@ -114,15 +114,15 @@ oc delete ns ${HC_CLUSTER_NS} || true
 # Validations
 export KUBECONFIG=${MGMT2_KUBECONFIG}
```
Comment (mburke5678) [inline modules/dr-hosted-cluster-within-aws-region-delete.adoc:115]:
```suggestion
$ export KUBECONFIG=${MGMT2_KUBECONFIG}
```

File: modules/dr-hosted-cluster-within-aws-region-delete.adoc [hunk 3]
```diff
@@ -34,8 +34,8 @@ As a workaround, update the value of the `spec.persistentVolumeClaimRetentionPol
 export KUBECONFIG=${MGMT_KUBECONFIG}
```
Comment (mburke5678) [inline modules/dr-hosted-cluster-within-aws-region-delete.adoc:34]:
```suggestion
$ export KUBECONFIG=${MGMT_KUBECONFIG}
```

File: modules/dr-hosted-cluster-within-aws-region-delete.adoc [hunk 4]
```diff
@@ -114,15 +114,15 @@ oc delete ns ${HC_CLUSTER_NS} || true
...
-oc get np -n ${HC_CLUSTER_NS}
-oc get pod -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME}
-oc get machines -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME}
+$ oc get hc -n ${HC_CLUSTER_NS}
+$ oc get np -n ${HC_CLUSTER_NS}
+$ oc get pod -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME}
+$ oc get machines -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME}
 
 # Inside the HostedCluster
 export KUBECONFIG=${HC_KUBECONFIG}
```
Comment (mburke5678) [inline modules/dr-hosted-cluster-within-aws-region-delete.adoc:123]:
```suggestion
$ export KUBECONFIG=${HC_KUBECONFIG}
```

**Comment by PoornimaSingour** [comment]
closing this as facing issue with SQuash , Updated the changes in https://github.com/openshift/openshift-docs/pull/82857

**Pull request closed without merging**